ignite/hatch.toml

```diff
@@ -0,0 +1,18 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["38"]
+version = ["2.8", "2.14"]
+
+[envs.default.overrides]
+matrix.version.env-vars = [
+    { key = "IGNITE_VERSION", value = "2.8.1", if = ["2.8"] },
+    { key = "IGNITE_IMAGE", value = "apacheignite/ignite:2.8.1", if = ["2.8"] },
+
+    { key = "IGNITE_VERSION", value = "2.14.0", if = ["2.14"] },
+    { key = "IGNITE_IMAGE", value = "apacheignite/ignite:2.14.0", if = ["2.14"] },
+]
+
+[envs.latest.env-vars]
```

Comment:
Are we still keeping `latest` as a valid environment?

Reply:
AFAIK yes, I moved latest from the ddev env to keep it and pinned the latest version for the environments